### DISCUSSION TITLE
How can I manager inventories holders ?

### DISCUSSION THREAD

**Discussion by Elikill58** [Minestom/Minestom#1353] [Help]
How can I manage holder as Bukkit has ?
Holder can correspond to player, but also to own object. I'm using it to check that it's my inventory and get data from it

**Comment by HSGamer** [comment]
I don't use Holder for checking custom inventories.

In my case, I extended the current [`Inventory`](https://javadoc.minestom.net/net/minestom/server/inventory/Inventory.html) to my own class with an [`EventNode<InventoryEvent>`](https://javadoc.minestom.net/net/minestom/server/event/EventNode.html) that checks if the inventory in the event is my inventory.
I can't give better explaination, but here is my implementation in my library HSCore: [`DelegatingInventory`](https://github.com/HSGamer/HSCore/blob/master/hscore-minestom/hscore-minestom-gui/src/main/java/me/hsgamer/hscore/minestom/gui/inventory/DelegatingInventory.java). Note that I created the `eventNode` and then register it to [`GlobalEventHandler`](https://javadoc.minestom.net/net/minestom/server/event/GlobalEventHandler.html) by using `MinecraftServer.getGlobalEventHandler().addChild(eventNode)`. All inventory events I want to register will be registered to that `eventNode`.
